curriculum/challenges/english/10-coding-interview-prep/rosetta-code/comma-quibbling.md

```diff
@@ -52,19 +52,19 @@ assert(typeof quibble(['ABC']) === 'string');
 assert.equal(quibble(testCases[0]), results[0]);
 ```
 
-`quibble(["ABC"])` should return "{ABC}".
+`quibble(["ABC"])` should return `"{ABC}"`.
 
 ```js
 assert.equal(quibble(testCases[1]), results[1]);
 ```
 
-`quibble(["ABC", "DEF"])` should return "{ABC and DEF}".
+`quibble(["ABC", "DEF"])` should return `"{ABC and DEF}"`.
 
 ```js
 assert.equal(quibble(testCases[2]), results[2]);
 ```
 
-`quibble(["ABC", "DEF", "G", "H"])` should return "{ABC,DEF,G and H}".
+`quibble(["ABC", "DEF", "G", "H"])` should return `"{ABC, DEF, G and H}"`.
```

Comment:
Sorry to drag this out even further - but this pointed out something to me, the test on line 79 is looking for `"{ABC,DEF,G and H}"` - if the function returns it with the spaces, it fails:
![Screen Shot 2022-06-16 at 10 32 07 AM](https://user-images.githubusercontent.com/20648924/174106487-db424469-7e2c-43a1-8933-b2136c4b622a.png)
 
The instructions say that spaces should be added. Do you mind adjusting that test real quick?